other/edge-server/client/react-client/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link
+      rel="icon"
+      href="/favicon.ico"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    />
+    <meta
+      name="theme-color"
+      content="#000000"
+    />
+    <meta
+      name="description"
+      content="A Todo List App built on MongoDB Atlas."
```

Comment:
Should this say something about Edge Server?

Reply:
Yeah, something like this, maybe?
```suggestion
      content="A Todo List App built with Atlas Edge Server."
```